src/test/run-make/long-linker-command-lines/Makefile

```diff
@@ -1,5 +1,5 @@
 -include ../tools.mk
 
 all:
-	$(RUSTC) foo.rs -g
+	$(RUSTC) foo.rs -g -O
```

Comment:
I found this test took *forever* without optimizations. It still takes a long time with them, but it's at least a little better